Tools/wasm/README.md

```diff
@@ -24,7 +24,7 @@ https://github.com/psf/webassembly for more information.
 To cross compile to the ``wasm32-emscripten`` platform you need
 [the Emscripten compiler toolchain](https://emscripten.org/), 
 a Python interpreter, and an installation of Node version 18 or newer.
-Emscripten version 3.1.73 or newer is recommended. All commands below are
+Emscripten version 4.0.2 or newer is recommended. All commands below are
 relative to a checkout of the Python repository.
```

Comment:
Given what we know from the buildbot, it seems appropriate to be explicit about how we test Emscripten.
```suggestion
Emscripten version 4.0.2 is recommended; newer versions may also work, but all
official testing is performed with that version. All commands below are relative
to a checkout of the Python repository.
```